services/requester/requester.go

```diff
@@ -470,6 +485,32 @@ func (e *EVM) replaceAddresses(script []byte) []byte {
 	return []byte(s)
 }
 
+// executeScriptAtHeight will execute the given script, at the given
+// block height, with the given arguments. A height of 0 is a special
+// value, which means the script will be executed at the latest sealed
+// block.
+func (e *EVM) executeScriptAtHeight(
+	ctx context.Context,
+	script []byte,
+	height uint64,
+	arguments []cadence.Value,
+) (cadence.Value, error) {
+	if height > 0 {
```

Comment:
you can't have other values than positive since the height is `uint`

Reply:
I would suggest defining our own special value for latest block that is something like:
```
latest := math.MaxUint64 -1 
```

Reply:
Right now, I am using `0` as a special case value. Also mentioned in the function documentation:
```go
// A height of 0 is a special value, which means the script will
// be executed at the latest sealed block.
```